sleap/gui/widgets/video.py

```diff
@@ -1732,6 +1733,211 @@ def updateEdge(self, node: QtNode):
             self.setLine(line)
 
 
+class VisibleBoundingBox(QtWidgets.QGraphicsRectItem):
+    """
+    QGraphicsRectItem for user instance bounding boxes.
```

Comment:
Follow [Google style docstrings](https://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings):
```suggestion
    """QGraphicsRectItem for user instance bounding boxes.
```
This allows us to see a one line description even when the docstring is collapsed.